src/api/environment.cc

```diff
@@ -436,6 +436,14 @@ Environment* GetCurrentEnvironment(Local<Context> context) {
   return Environment::GetCurrent(context);
 }
 
+IsolateData* GetEnvironmentIsolateData(Environment* env) {
+  return env->isolate_data();
+}
+
+ArrayBufferAllocator* GetArrayBufferAllocator(IsolateData* isolate_data) {
+  return isolate_data->node_allocator();
```

Comment:
`node_allocator` is public in `IsolateData` instance, so it is something that the caller is able to do itself?

Reply:
But `IsolateData` itself is not public.